### PR TITLE
Update readme with installation and usage instructions

### PR DESCRIPTION
**Pull request by p-j-smith**
- update readme with installation and usage instructions
- also add code style, license, and author info

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,145 @@
...
+```
+
+### Run the tests using Molecule
+
+Molecue 6.0 requires that the test configuration is not in the top-level directory of the
+collection. To support running the tests with Molecule 6, the Molecule configuration is in
+`ansible_collections/mirsg/infrastructure/tests`. To run the tests you must be in this directory:
+
+```
+cd ansible_collections/mirsg/infrastructure
```
Comment (drmatthews) [inline README.md:65]:
```suggestion
cd ansible_collections/mirsg/infrastructure/tests
```

File: README.md [hunk 2]
```diff
@@ -1,3 +1,145 @@
...
+molecule playbooks. If the deployment fails, the container is not destroyed.
+
+#### Access the container
+
+Once the command has finished running, you can access the container using its
+[names defined in the scenario](tests/molecule/centos7/molecule.yml). To access the
+container (named `instance`):
+
+```
+docker exec -it instance /bin/bash
```
Comment (drmatthews) [inline README.md:112]:
```suggestion
molecule login -s centos7
```

**Review by drmatthews — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (p-j-smith) [reply README.md:112]: ooo I didn't know you could do that!

Comment (p-j-smith) [reply README.md]: I've added your suggestion and changed the text directly above it to reflect the change

File: README.md (hunk 1 above)
Comment (p-j-smith) [reply README.md]: well spotted!

**Pull request merged**